templates/server/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -6,6 +6,9 @@
   <id>devon4j-template-server</id>
 
   <requiredProperties>
+    <requiredProperty key="version">
+            <defaultValue>${projectVersion}</defaultValue>
```

Comment:
your indendation seems to be broken. What IDE are you using? should be 2 spaces per indendation level (and not tabs)

Reply:
Where is the `version` variable used? I do not entirely get this idea.

Reply:
This part has been removed now.